### ISSUE TITLE
Request for "Mongoose" Icon

### ISSUE THREAD

**Issue by PIYAS79** [tandpfun/skill-icons#1006]
### Skill Name

Mongoose 

### Why?

Mongoose is a vastly used technology but sad to say I used your icon for 2 years but I can't find the mongoose icon in your repo !!! It really ........!!! I think you understand!

### Reference Image

(https://camo.githubusercontent.com/55633046c00899f6c7dc74ca280c7313cfe632f156c4fa50841428415a3ec2d5/68747470733a2f2f63646e2e6a7364656c6976722e6e65742f67682f64657669636f6e732f64657669636f6e406c61746573742f69636f6e732f6d6f6e676f6f73652f6d6f6e676f6f73652d6f726967696e616c2d776f72646d61726b2e737667)

**Comment by Janethq** [comment]
Agree with this! Lets have mongoose please!

**Comment by LelouchFR** [comment]
@PIYAS79 @Janethq I might try to do something about it.

> [!NOTE]
> There is a new version of the repository, because the maintainer vanished from here... so we've moved on and did a lot of changes and we are already at around 450 icons and you might find other icons that could help you out. The repository is [here](https://github.com/lelouchfr/skill-icons) is you want to check it and get informations about what could be worked on and other things

**Comment by PIYAS79** [comment]
A little sooner would have helped a lot!

Thank you,
S M Piyas Mahamude Alif

**Comment by LelouchFR** [comment]
@PIYAS79 @Janethq what do you guys think about that ?

![image](https://github.com/tandpfun/skill-icons/assets/75901978/3fbf9bf8-2929-4188-bcbf-9ab37a885047)

**Comment by PIYAS79** [comment]
The white one is more attractive than the black one !!!

You should submit the white one if my other friend @Janethq agrees!
but personally, I prefer the white one !!!


Thanks From, 
S M Piyas Mahamude Alif

**Comment by LelouchFR** [comment]
The thing is that it should be dark & light or only one, if it should only be one, what do you think about that ?

![image](https://github.com/tandpfun/skill-icons/assets/75901978/f7fe916f-be75-49dd-b524-cf616b5637c9)

**Comment by PIYAS79** [comment]
I think white looks more beautiful and meaningful because the color of the content inside the white matches the real logo very well !!

This is just my personal opinion!

from, 
S M Piyas Mahamude Alif

**Comment by LelouchFR** [comment]
> I think white looks more beautiful and meaningful because the color of the content inside the white matches the real logo very well !!
> 
> This is just my personal opinion!
> 
> from, S M Piyas Mahamude Alif

yeah, but afetr that there's some problem with the dark icon, which looks not good at all I think, the icon I've made with the red background fits the red of the logo and I think it would be all good like that. for example, the golang logo is also normally blue, but in this case, the blud is put on the background for some better comprehension. I think I would stick with the red background and white logo for better readabillity.

**Comment by PIYAS79** [comment]
I agree with you!!


Greeting,

Piyas

**Comment by Janethq** [comment]
> @PIYAS79 @Janethq what do you guys think about that ?
> 
> ![image](https://private-user-images.githubusercontent.com/75901978/344224336-3fbf9bf8-2929-4188-bcbf-9ab37a885047.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTk2OTA1ODEsIm5iZiI6MTcxOTY5MDI4MSwicGF0aCI6Ii83NTkwMTk3OC8zNDQyMjQzMzYtM2ZiZjliZjgtMjkyOS00MTg4LWJjYmYtOWFiMzdhODg1MDQ3LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA2MjklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNjI5VDE5NDQ0MVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTdkYmEyYWMwZWJjMmJhMjcwYmFhNGI5NjZiOWM0MTM4NjlmZGIyNTA2MDlmMzExMzVjZjFlOTQ4OGY1YzBmZmMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.qFgopb9A1KAbTPatBnlVnc1Qd0LuRKkIBjb-pMsKE7k)

I LOVE THE WHITE. Thank you!

**Comment by Janethq** [comment]
> > I think white looks more beautiful and meaningful because the color of the content inside the white matches the real logo very well !!
> > This is just my personal opinion!
> > from, S M Piyas Mahamude Alif
> 
> yeah, but afetr that there's some problem with the dark icon, which looks not good at all I think, the icon I've made with the red background fits the red of the logo and I think it would be all good like that. for example, the golang logo is also normally blue, but in this case, the blud is put on the background for some better comprehension. I think I would stick with the red background and white logo for better readabillity.

Oh yeah I understand your considerations.

**Comment by PIYAS79** [comment]
@LelouchFR   heyy !!!

You haven't uploaded the icon yet   !!!??
I am very excited to add that icon to my profile !!!!

**Comment by PIYAS79** [comment]
Hiiiiiiiiiiiiiiiiii @LelouchFR  ........!!!

I still have not found the Mongoose icon !!!

**Comment by PIYAS79** [comment]
@LelouchFR ........!!!

I still have not found the Mongoose ic000000n

![310m5r](https://github.com/user-attachments/assets/6934e7df-4502-4e03-bad0-a4b81773a2ed)
